cmd/monaco/download/download_configs.go

```diff
@@ -256,6 +256,18 @@ func downloadConfigs(downloaders downloaders, opts downloadConfigsOptions) (proj
 		}
 	}
 
+	{
+		if opts.auth.OAuth != nil {
+			bucketCfgs, err := downloaders.Bucket().Download(opts.projectName)
```

Comment:
```suggestion
			log.Info("Downloading Grail buckets")
			bucketCfgs, err := downloaders.Bucket().Download(opts.projectName)
```

please add a log to match the others